tests/bdd-smoke/Gopkg.toml

```diff
@@ -0,0 +1,15 @@
+[[constraint]]
```

Comment:
Can you break out the dependency imports into another commit? It just makes it a little easier to follow the history.

Reply:
Done